From 43cc1fcd05b488dc588af843308c1ab2c08e0772 Mon Sep 17 00:00:00 2001
From: Guoliang Ji <guoliang.ji@intel.com>
Date: Thu, 4 Jul 2013 10:26:16 +0800
Subject: refine the profile setting

BZ:101039

Refine the profile setting to enable the high profile encoding.

Category: feature differentiation
Domain: Video.Media-video recording
Origin: internal
Upstream-Candidate: No

Change-Id: Id5e638c8e69848e4ec9680ea6975c4ee8df01bc7
Orig-Change-Id: I8badca7325e7b2863a3fdc6949e5424080ec163e
Signed-off-by: jiguoliang <guoliang.ji@intel.com>
Signed-off-by: ywan171 <yi.a.wang@intel.com>
Signed-off-by: Dan Liang <dan.liang@intel.com>
---
 media/libstagefright/OMXCodec.cpp |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/media/libstagefright/OMXCodec.cpp b/media/libstagefright/OMXCodec.cpp
index e54db65..de0560c 100644
--- a/media/libstagefright/OMXCodec.cpp
+++ b/media/libstagefright/OMXCodec.cpp
@@ -1273,13 +1273,6 @@ status_t OMXCodec::setupAVCEncoderParameters(const sp<MetaData>& meta) {
     h264type.eProfile = static_cast<OMX_VIDEO_AVCPROFILETYPE>(profileLevel.mProfile);
     h264type.eLevel = static_cast<OMX_VIDEO_AVCLEVELTYPE>(profileLevel.mLevel);
 
-    // XXX
-    if (h264type.eProfile != OMX_VIDEO_AVCProfileBaseline) {
-        ALOGW("Use baseline profile instead of %d for AVC recording",
-            h264type.eProfile);
-        h264type.eProfile = OMX_VIDEO_AVCProfileBaseline;
-    }
-
     if (h264type.eProfile == OMX_VIDEO_AVCProfileBaseline) {
         h264type.nSliceHeaderSpacing = 0;
         h264type.bUseHadamard = OMX_TRUE;
-- 
1.7.4.1

